src/search/merge_and_shrink/fts_factory.cc

```diff
@@ -420,9 +443,9 @@ vector<unique_ptr<Distances>> FTSFactory::create_distances(
     assert(num_variables >= 1);
     result.reserve(num_variables * 2 - 1);
 
-    for (int var_no = 0; var_no < num_variables; ++var_no) {
+    for (int var_id = 0; var_id < num_variables; ++var_id) {
```

Comment:
rename to `var`?

Reply:
For this and other places: we consistently use `var` for `VariableProxy` and `var_id` for the int retrieved from `VariableProxy::get_id`, so I would like to stick with `var_id` here and at other analogous places.